### PR TITLE
Document routes and their query parameters

### PR DESCRIPTION
**Pull request by imujjwal96**

### CODE REVIEW

**Comment by imujjwal96** [comment]
@syedsaadh  Please review.

File: server.js [hunk 1]
```diff
@@ -54,7 +70,16 @@ app.get('/:width/:height/:bg', function (req, res) {
     drawString(ctx, req.query.txt || 'Awesome! \n Library', '#' + (req.query.textColor || 'fff'), 0, req.query.font, req.query.fontSize, width, height);
     res.writeHead(200, { 'Content-Type': 'image/png' })
     canvas.pngStream().pipe(res)
-})
+});
+
+/**
+ * Generate an avatar image
```
Comment (syedsaadh) [inline server.js:78]: Why following two standards for Routes Help. Either use 
``` req.params```
```- :width: width of the banner (in pixels)```
Or
``` req.params.width: width of the banner (in pixels)```

**Review by syedsaadh — CHANGES_REQUESTED** [review]
Please go through suggestions

File: server.js (hunk 1 above)
Comment (imujjwal96) [reply server.js:78]: makes sense. Will change it.

**Comment by imujjwal96** [comment]
@syedsaadh I've updated it

**Review by syedsaadh — APPROVED** [review]

**Comment by syedsaadh** [comment]
@imujjwal96 Great. And Thanks for PR

**Pull request merged**